### ISSUE TITLE
Todos Removed From Dashboard

### ISSUE THREAD

**Issue by fmcpherson** [monicahq/monica#803]
Beginning with release 1.5 Todo items no longer appear on the Dashboard, making it harder to track the Todo items. I recommend adding Todos to the box where Calls and Important Notes display on the dashboard. 

A feature enhancement could be adding a Tasks section (like you have for Journal) that displays a checklist of all the consolidated Todo items.

**Comment by djaiss** [comment]
Indeed, that's something we've forgotten to include in the redesign. We need to fix that.

**Comment by rtpg** [comment]
@djaiss have you started working on this. I was looking at maybe adding non-completed tasks into the dashboard log?

**Comment by fmcpherson** [comment]
I see this topics is still open. We now have completed Todos on the dashboard, however, I would like a way to filter out or remove completed tasks from the dashboard, particularly if you are going to complete displaying contact tasks. My completed task list is getting long.
